build/cpp/kvstore/src/kvstore_test.cc

```diff
@@ -20,15 +19,68 @@ TEST(KVStore, PuttingMultiple) {
   EXPECT_EQ((*kvstore.Get("key1"))[2], "value3");
 }
 
-// test put then remove
+// put then remove
 TEST(KVStore, PutAndRemove) {
-  kvstore::KVStore kvstore;
+  kvstore::KVStore kvstore("");
   kvstore.Put("key1", "value1");
   EXPECT_EQ((*kvstore.Get("key1"))[0], "value1");
   kvstore.Remove("key1");
   EXPECT_EQ(kvstore.Get("key1"), std::nullopt);
 }
 
+// write file on existing file
+TEST(KVStore, ExistingWriteFile) {
+  kvstore::KVStore kvstore("test1.txt");
+  kvstore.Put("key1", "value1");
+  kvstore.Put("key1", "value2");
+  kvstore.Put("key2", "value3");
+  kvstore.WriteFile();
+  kvstore.WriteFile();
```

Comment:
How are these tests indicating success/failure?

Reply:
Tests have been updated to better indicate success/failure.